metadata/db-xrefs.yaml

```diff
@@ -2066,9 +2066,9 @@
     - type_name: protein
       type_id: PR:000000001
       id_syntax: ([OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2}){1}(-[0-9]+){0,1}|[0-9]{9}
-      url_syntax: http://purl.obolibrary.org/obo/PR_[example_id]
+      url_syntax: https://purl.obolibrary.org/obo/PR_[example_id]
```

Comment:
Hi @balhoff

I have to admit I didn't think deeply about this before I pushed the change.  We use `url_syntax` to add links on pombase.org pages.  We prefer HTTPS so that users don't get browser warnings when they click.

If HTTP makes more sense that's no problem for PomBase.  We can change the purl URLs to HTTPS when we read db-xrefs.yaml

Reply:
@balhoff The purpose of this file from the GO POV is _not_ to maintain identifiers, but rather linkouts via CURIEs. Unfortunately, I suspect that shortcuts may have been taken various places.
However, as AmiGO is now HTTPS, we're going to have to start upgrading entires as necessary to HTTPS as that's what browsers are increasingly requiring.

@kimrutherford Apologies that this slipped through. If you still want this in, tag me and I'll merge.

Reply:
Alright, https makes sense then!